ui/marketplace/useMarketplace.tsx

```diff
@@ -74,7 +74,12 @@ export default function useMarketplace() {
   const { isPlaceholderData, isError, error, data, displayedApps } = useMarketplaceApps(debouncedFilterQuery, selectedCategoryId, favoriteApps);
 
   const categories = React.useMemo(() => {
-    return _unique(data?.map(app => app.categories).flat()) || [];
+    const grouped = _groudBy(data, app => app.categories);
+    return Object.keys(grouped).map(category => ({
+      name: category,
+      count: grouped[category].length,
+    }));
+    // return _unique(data?.map(app => app.categories).flat()) || [];
```

Comment:
Let's remove this line because it is unnecessary.

Reply:
I left that line in just in case, but in the following PR it is finally removed: https://github.com/blockscout/frontend/pull/1545/files#diff-2ebed5c8ee6bd0ee84fe6b654067cdee18307a229868afd84e6652d64a866517L82